build-headless-shell.sh

```diff
@@ -197,7 +197,7 @@ for i in $(seq 1 $ATTEMPTS); do
   RET=1
   echo "STARTING BUILD ATTEMPT $i FOR $VERSION ($(date))"
   #$SRC/icecc-chromium/icecc-ninja -j $JOBS -C $PROJECT headless_shell chrome_sandbox && RET=$?
-  $SRC/depot_tools/ninja -j $JOBS -C $PROJECT headless_shell chrome_sandbox && RET=$?
```

Comment:
`chrome_sandbox` is not included in the final image. It seems that the builder server does not use the same `build-headless-shell.sh` in this repository.